pkg/collector/corechecks/net/network/network.go

```diff
@@ -71,6 +71,15 @@ var (
 	udpStateMetricsSuffixMapping = map[string]string{
 		"NONE": "connections",
 	}
+
+	filesystem = afero.NewOsFs()
+
+	ethtoolObject = ethtool.Ethtool{}
+	getDrvInfo    = ethtoolObject.DriverInfo
+	getStats      = ethtoolObject.Stats
```

Comment:
```suggestion
	getEthtoolDrvInfo    = ethtoolObject.DriverInfo
	getEthtoolStats        = ethtoolObject.Stats
```

To improve the naming

Reply:
fixed in https://github.com/DataDog/datadog-agent/pull/33190/commits/7ac3c6bc005e18f249afbcbbe44eedec4b26ee27